app/frontend/actions/delegate.ts

```diff
@@ -131,10 +144,71 @@ export default (store: Store) => {
     })
   }
 
+  const delegate = async (state: State): Promise<void> => {
+    return await confirmTransaction(state, {
+      txConfirmType: TxType.DELEGATE,
+      txPlan: state.cachedTransactionSummaries[TxType.DELEGATE].plan,
+      sourceAccountIndex: state.sourceAccountIndex,
+    })
+  }
+
+  const deregisterStakingKey = async (state: State): Promise<void> => {
```

Comment:
I find it a bit odd that the file grouping these actions is called delegate - if it tries to group actions related to staking, perhaps that would be a more descriptive name, but I understand that would trigger quite a lot of changes, so this is rather a note for a later iteration of refactoring

Reply:
Thanks, I could create an issue for that, so it can be changed in standalone PR.